### DISCUSSION TITLE
Non recursive search

### DISCUSSION THREAD

**Discussion by manfredlotz** [BurntSushi/ripgrep#1833] [Q&A] [answered]
I like to know if there is a possibility to do a non recursive search. Didn't find anything, perhaps I overlooked it.

**Comment by BurntSushi** [comment, marked as answer]
You probably want the `--max-depth` flag.

**Comment by manfredlotz** [comment]
Yes, thanks. I just didn't see it. Searched for recursive and such.  Perhaps, it would be nice to have a short flag for `--max-depth 1`.

**Reply by BurntSushi** [reply]
Short flags are unfortunately pretty scarce real estate. `--max-depth` isn't a terribly common flag, so it's unlikely to get a short alias. Adding the word "recursive" to the docs would be a good idea though.

**Reply by manfredlotz** [reply]
> Short flags are unfortunately pretty scarce real estate.

Unfortunately, you are right. :-)
